src/test/addresses_deployers.sol

```diff
@@ -14,13 +14,13 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity 0.6.12;
+pragma solidity 0.8.16;
```

Comment:
on `spells-goerli` mirror PR `^` is used, not sure we want to added here as well

Reply:
There is a mix in spells-goerli, we need to fix that.